src/lib/tls/tls_policy.cpp

```diff
@@ -127,6 +127,17 @@ size_t Policy::minimum_dh_group_size() const
    return 1024;
    }
 
+size_t Policy::minimum_ecdh_group_size() const
+   {
+   return 159;
```

Comment:
With 1024 bit RSA keys, we experienced problems that some servers generated 1023, 1022...etc bit keys, and lost some security. To stay compatible, we set it then to 1000 bits.

We assume this can also happen in case of EC. We wanted to set the minimum size to 160 bits.
Do you mean this behaviour cannot occur in case of EC?
What should be then the default value then? What do you propose?